cli/azd/cmd/testdata/TestUsage-azd-down.snap

```diff
@@ -1,6 +1,9 @@
 Usage:
   azd down [flags]
 
+Aliases:
+  down, infra delete
```

Comment:
Is this showing up correctly? `down` is listed as an alias for `down`..

Reply:
Seems to be affecting other aliases as well.

Reply:
That must be a cobra thing - I don't know why it reports itself as an alias when it is not passed.